### PR TITLE
This PR fixes issue #41

### PR DESCRIPTION
**Pull request by dimiii**
Influxdb at least since 1.4.2 does not process queries like 'select * from dummy where 1 = 0'

Please, note also as I changed tests for the negation of the none-expression.

### CODE REVIEW

File: lib/influxer/metrics/relation/where_clause.rb [hunk 1]
```diff
@@ -59,7 +59,7 @@ def build_eql(key, val, negate)
       when Regexp
         "#{key}#{negate ? ' !~ ' : ' =~ '}#{val.inspect}"
       when Array
-        return build_none if val.empty?
+        return build_none(negate) if val.empty?
```
Comment (palkan) [inline lib/influxer/metrics/relation/where_clause.rb]: That would be a breaking change, unfortunately( But that's the right way to handle empty values.

Please, add a change log entry with "breaking" in **bold**

**Comment by dimiii** [comment]
Done, could you release it?

File: Changelog.md [hunk 2]
```diff
@@ -1,5 +1,14 @@
 # Change log
 
+## 1.1.6
+
+- [Fixes [#41](https://github.com/palkan/influxer/issues/41)] Fix query building with empty arrays in `where` clause ([@dimiii][])
+
+  [PR](https://github.com/palkan/influxer/pull/44)
+
+  **BREAKING:** So far as the latest versions of influxdb fail on queries like `select * from dummy where 1 = 0`, instead of
```
Comment (palkan) [inline Changelog.md]: The breaking change is that `where.not` now returns non-empty result (and not the internal implementation)

Comment (dimiii) [reply Changelog.md]: Fixed.

**Comment by palkan** [comment]
Released

**Comment by dimiii** [comment]
Thank you!
